decidim-design/README.md

```diff
@@ -0,0 +1,29 @@
+# decidim-design
+
+The DESIGN module implements the design guide for Decidim, a guide that includes all information related with the new design of the platform.
```

Comment:
Consistency with the rest of the modules explanation: 
```suggestion
The Design module implements the design guide for Decidim, a guide that includes all information related with the design of the platform.
```

Also now it's new, but as it'll be just the design, let's drop the word.